python/dgl/data/citation_graph.py

```diff
@@ -29,6 +30,7 @@ def _pickle_load(pkl_file):
 
 class CitationGraphDataset(object):
```

Comment:
Some docstring about the tensors stored in the returned DGLGraph? Example codes are also appreciated. Similar for the other datasets.

Reply:
added